src/pages/AccessResultRecords.js

```diff
@@ -180,8 +181,14 @@ class AccessResultRecords extends Component {
           div({ className: "col-lg-10 col-md-9 col-sm-9 col-xs-12 no-padding" }, [
             PageHeading({
               id: "recordAccess", imgSrc: "/images/icon_access.png", iconSize: "medium", color: "access",
-              title: "Data Access - Results Record", description: consentData
-            }),
+              title: "Data Access - Results Record"}),
+              DarDetails({
+                projectTitle: this.state.projectTitle,
+                darCode: this.state.darInfo.darCode,
+                datasetId: this.state.darInfo.datasetId,
+                datasetName: this.state.darInfo.datasetName,
+                consentName: this.state.consentName
```

Comment:
In testing, I'm seeing undefined values here.

Reply:
I see what's happening here. `consentName` is not a state variable on `AccessResultRecords`, but it is on `AccessCollect`.